README.md

```diff
@@ -31,9 +31,29 @@ There are three levels of configuration needed in order for publishing and fetch
    - Specify the store id
    - Choose the docker registry artifact plugin
    - Specify the `Docker Registry Url`
-   - Specify `username` and `password`
+   - Specify the `Registry Type`
+   - Specify registry details
+
+   If `Registry Type` is `ecr`:
+
+   ![](images/artifact_store_ecr.png)
+
+   If `Registry Type` is `other`:
+
+   ![](images/artifact_store_other.png)
+
+###### Artifact Store Properties
+
+| Properties                     | Description                                                              | Required                  |
+| -------------------------------| ------------------------------------------------------------------------ | --------------------------|
+| `RegistryURL`                  | Registry to push and pull images                                         | `true`                    |
+| `RegistryType`                 | Can be either `ecr` or `other`                                           | `true`                    |
+| `Username`                     | The size of the PVC                                                      | `true` if type is `other` |
+| `Password`                     | The PVC storage class name                                               | `true` if type is `other` |
```

Comment:
Instead of `Username` and `Password` perhaps change the label to `access key id` and `secret access key` if the type is `ecr`, hopefully that reduces the number of toggles?

Reply:
the documentation can clarify what actually goes into these fields